src/components/Pages/WithText/Buttons/ToggleScan/AddScanBookmarkBtn.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Modal } from 'semantic-ui-react';
+import { useSelector } from 'react-redux';
+
+import BookmarkForm from '../../../../shared/SaveBookmark/BookmarkForm';
+import { selectors as settings } from '../../../../../redux/modules/settings';
+import ScanBtnTpl from './ScanBtnTpl';
+
+const urlProps           = { 'scan': true };
+const AddScanBookmarkBtn = () => {
+  const { t } = useTranslation();
+  const dir   = useSelector(state => settings.getUIDir(state.settings));
+
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+
+  const handleClose = (e, el, isCreated) => {
+    if (!isCreated) {
```

Comment:
Why do you need this `if`? Both inside it and outside you're calling `setOpen(false)`

Reply:
fixed